src/templates/wrappers/description.ts

```diff
@@ -11,5 +11,5 @@ import {FieldWrapper} from '../field.wrapper';
   `,
 })
 export class FormlyWrapperDescription extends FieldWrapper {
-  @ViewChild("fieldComponent", {read: ViewContainerRef}) fieldComponent: ViewContainerRef;
+  @ViewChild('fieldComponent', {read: ViewContainerRef}) fieldComponent: ViewContainerRef;
 }
```

Comment:
@mohammedzamakhan @aitboudad do we plan to use single quote or double quote? The rules in codacy and tslint.json may not be consistent.